junit-extension/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.togglz</groupId>
+    <artifactId>togglz-project</artifactId>
+    <version>2.6.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>togglz-junit-extension</artifactId>
```

Comment:
I would prefer the artifact name `togglz-junit5`. Having `togglz-junit` and `togglz-junit-extension` would be confusing. But I think if we have `togglz-junit` and `togglz-junit5`, people will understand the difference. Please also update the name of the directory to `junit5`.